<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgPackageExporter</class>
 <widget class="QDialog" name="dlgPackageExporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="filePath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>491</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Path</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>561</width>
     <height>241</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Check Items To Export</string>
    </property>
   </column>
   <item>
    <property name="text">
     <string>Triggers</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Aliases</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Timers</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Scripts</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Keys</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Buttons</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgPackageExporter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgPackageExporter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
